entry_tasks/16_Pick_movie_by _genre_or_actor/by_genre_and_cast.py

```diff
@@ -0,0 +1,119 @@
+def unique_cast(cast_values):
+    unique_cast_list = []
+    for x in cast_values:
+        if x not in unique_cast_list:
+            unique_cast_list.append(x)
+    return unique_cast_list
+
+
+def unique_movies(genres_data_values):
+    unique_list1 = []
+    for i in genres_data_values:
+        if i not in unique_list1:
+            unique_list1.append(i)
+    return unique_list1
+
+
+def search_by_genre():
```

Comment:
this functions does not make a lot of sense. you've just copied code snippets into the function definitions.

From testing purposes:
1. Function should do only one thing. Accept value, return result. Even if you print result to user, it still a result which function should return (all input and print should happen in main block), but some search can be moved into a function.
2. Function should not be huge if possible, try to find common things between search_by_genre and search_by_cast if there are any and move them into a functions only if possible. Because right now you have spaggetti one time functions here :)